packages/wrangler/src/dev.tsx

```diff
@@ -782,28 +782,39 @@ function useHotkeys(initial: useHotkeysInitialState, port: number) {
       key
     ) => {
       switch (input) {
```

Comment:
It's a bit simpler (and less to maintain) if you do:

```suggestion
      switch (input.toLowerCase()) {
```